packages/botonic-cli/tests/util/file-system.test.ts

```diff
@@ -0,0 +1,123 @@
+import { readFileSync, writeFileSync } from 'fs'
+import { join } from 'path'
+
+import {
+  copy,
+  create,
+  createTemp,
+  pathExists,
+  readDir,
+  readJSON,
+  remove,
+  writeJSON,
+} from '../../src/util/file-system'
+import { execCommand } from '../../src/util/processes'
+
+const existingPath = process.cwd()
+const unexistingPath = 'unexistingPath'
+const dirToTest = join(process.cwd(), 'botonic-tmp')
+
+const withDirToTest = (
+  toTest: () => any,
+  { createDir } = { createDir: true }
+): boolean => {
+  let success = false
```

Comment:
in many usages you don't check the returned boolean, and hence you won't realize if toTest() fails. Also, you won't even see any error callstack.
I suggest 2 alternatives:
* creating/removing the dir in beforeEach/afterEach
* Remove the try/catch and manage the expected faiures wih a expect/toThrow

In general, always first try using the tools provided by your framework.